server/models/services/TestResultReadService.js

```diff
@@ -0,0 +1,86 @@
+const unexpectedBehaviorsJson = require('../../resources/unexpectedBehaviors.json');
+const getTests = require('./TestsService');
+const AtLoader = require('../loaders/AtLoader');
+const BrowserLoader = require('../loaders/BrowserLoader');
+const deepCustomMerge = require('../../util/deepCustomMerge');
+
+/**
+ *
+ */
+const getTestResults = async testPlanRun => {
+    const { testPlanReport } = testPlanRun;
+    const tests = getTests(testPlanReport);
+    const atLoader = AtLoader();
+    const browserLoader = BrowserLoader();
```

Comment:
I suppose so. The only reason it is split out into multiple lines like this is to better match how the `atLoader` and `browserLoader` were passed around as single instances previously. The tradeoffs is that there were several places where I was able to remove `context` entirely, and avoid things like `serviceFunction(id, null, null, context);`.

Pound for pound the singleton removes more characters than it adds and more importantly, it allows services to be used outside of a graphql resolver which is required for automation. I'd be open to looking for a more ergonomic approach but I wonder if this might be better done as a follow-up since it will be hard for reviewers to see tradeoffs for a change like this in the context of such a big PR.